dds/idl/langmap_generator.cpp

```diff
@@ -1407,6 +1407,12 @@ struct Cxx11Generator : GeneratorBase
     }
     be_global->add_include("<array>", BE_GlobalData::STREAM_LANG_H);
     be_global->lang_header_ << ind << "using " << type << " = " << array << elem << bounds.str() << ";\n";
+
+    if (!be_global->suppress_typecode()) {
+      be_global->add_include("tao/Basic_Types.h", BE_GlobalData::STREAM_LANG_H);
+      be_global->lang_header_ << "extern ::CORBA::TypeCode_ptr const _tc_" << type << ";\n";
```

Comment:
```suggestion
      be_global->lang_header_ << "extern const ::CORBA::TypeCode_ptr _tc_" << type << ";\n";
```
Same on the next line.
Since the same block appears 4 times, refactor to a common function.